Gruntfile.js

```diff
@@ -173,9 +173,18 @@ module.exports = function(grunt) {
           zip: "dist/chrome.zip"
         }
       }
+    },
+
+    eslint: {
+      target: ["app/scripts/**/*.js"],
+      options: {
+        fix: true
```

Comment:
honestly does scare me a little bit... but i have faith it isn't too invasive. i'm guessing you ran this over the entire project? i've sort of skimmed through and it seems to have just added white spaces, triple equals, and cleaned up unused arguments/vars.